pages/events/index.vue

```diff
@@ -83,9 +83,38 @@
           </div>
         </template>
         <template slot="extra-info">
-          <div class="event-page__start-an-event">
+          <div class="event-page__section">
+            <h3>Follow our event calendar</h3>
+            <p class="event-page__section__description">
+              <!-- eslint-disable-next-line vue/singleline-html-element-content-newline -->
+              Stay up to date with all of our scheduled events by following our calendar. You can view the calendar by visiting <AppLink v-bind="qiskitCalendarLink">{{ qiskitCalendarLink.label }}</AppLink>, or subscribe to it by adding to the calendar app of your choice.
+            </p>
+            <div class="event-page__tabs">
+              <cv-tabs aria-label="navigation tab label">
+                <cv-tab
+                  v-for="(calendar, calendarIndex) in calendars"
+                  :id="`tab-${calendar.name}`"
+                  :key="calendarIndex"
+                  :label="calendar.name"
+                  class="event-page__tab"
+                >
+                  <p class="event-page__sync">
+                    Start by copying the calendar subscription link
+                    <cv-code-snippet
+                      kind="inline"
+                      feedback-aria-label="Copy calendar sync url"
+                    >
+                      <span>{{ qiskitCalendarSyncLink }}</span>
+                    </cv-code-snippet>
+                  </p>
+                  <component :is="calendar.component" class="event-page__instructions" />
```

Comment:
For this particular case, I think that having the tabs in this component might be easier to maintain instead of creating 3 new components with very little content and no logic or style.

If we want to do this, we could achieve it like this:

```suggestion
                  <div class="event-page__instructions">
                    <template v-if="calendar.name === 'Google'">
                      <cv-list ordered>
                        <cv-list-item>
                          Open your Google Calendar by visiting <AppLink url="https://calendar.google.com/calendar/u/0/r/settings/addbyurl?pli=1">
                            this link
                          </AppLink>
                        </cv-list-item>
                        <cv-list-item>Paste the events subscription link you copied above</cv-list-item>
                        <cv-list-item>Click "Add Calendar"</cv-list-item>
                      </cv-list>
                    </template>
                    <template v-else-if="calendar.name === 'Outlook'">
                      <cv-list ordered>
                        <cv-list-item>Open your Outlook Calendar and click on “Add calendar”</cv-list-item>
                        <cv-list-item>Select the menu item “Subscribe from web”</cv-list-item>
                        <cv-list-item>Paste the events subscription link you copied above</cv-list-item>
                        <cv-list-item>Add a name for your calendar</cv-list-item>
                        <cv-list-item>Click “Import”</cv-list-item>
                      </cv-list>
                    </template>
                    <template v-else>
                      <cv-list ordered>
                        <cv-list-item>Open your Apple Calendar</cv-list-item>
                        <cv-list-item>In the menu, select File > New Calendar Subscription</cv-list-item>
                        <cv-list-item>Paste the events subscription link you copied above</cv-list-item>
                        <cv-list-item>Click "Subscribe"</cv-list-item>
                        <cv-list-item>Click "Ok"</cv-list-item>
                      </cv-list>
                    </template>
                  </div>
```

Reply:
FYI, had a discussion w/ @y4izus on this, and decided to keep the calendar instructions separate from this page component. I agree that there's little content w/ no logic or style... but we'll still have these benefits

- easy to find/maintain, since it's cleanly separated into their own files
- doesn't pollute the parent page (which will continue to grow as we potentially add the actual calendar as well)

The main ask was to rename the component and restructure how they were organized, done here https://github.com/Qiskit/qiskit.org/pull/2955/commits/2308580cf6a79698c60a1279bca8f12fb85995de